src/stactools/sentinel1/grd/properties.py

```diff
@@ -0,0 +1,89 @@
+from stactools.core.io.xml import XmlElement
+from pystac.extensions.sar import FrequencyBand, Polarization
+from pystac.extensions.sat import OrbitState
+
+
+def fill_sar_properties(sar_ext, href):
+    """Fills the properties for SAR.
+
+    Based on the sar Extension.py
+
+    Args:
+        input_ext (pystac.extensions.sar.SarExtension): The extension to be populated.
+        href (str): The HREF to the scene, this is expected to be an XML file.
+
+    Returns:
+        pystac.Asset: An asset with the SAR relevant properties.
+    """
+    # Read meta file
+    root = XmlElement.from_file(href)
+
+    # Fixed properties
+    sar_ext.frequency_band = FrequencyBand("C")
+    sar_ext.center_frequency = 5.405
```

Comment:
in the future would be good to consolidate common properties for GRD,RTC,SLC, etc under `sentinel1/properties.py`